mobile-browser-based-version/src/logic/communication/decentralised/decentralised_client.ts

```diff
@@ -158,21 +160,17 @@ export class DecentralisedClient extends Client {
   /**
    * Disconnection process when user quits the task.
    */
-  disconnect () {
+  async disconnect () {
```

Comment:
we can discuss the `async`/`await` policy later on @Nacho114

Reply:
yes, this is important, it's not clear to me what it is right now